src/js/helpers/cleanData.js

```diff
@@ -720,3 +720,19 @@ export const cleanFormContainersForPreview = allFormContainers => {
 
   return form;
 };
+
```

Comment:
explanation about what this is for would be great; it seems kinda tangental to the events page stuff but aimed at cleaning up some stuff for previews & IA stuff?

Reply:
everything up there was just jsprettier formatting. The getOfferedByFromRelatedDepartments logic used to be hiding around a bunch of places in some form or another.

We use it in static.config.js, but previews don't use the logic in there, so I added it in.

We don't have the cleanest solutions around Janis for this stuff, and I don't have any good ideas for ways to clean it up at the moment